dockstore-webservice/src/main/java/io/dockstore/webservice/resources/UserResource.java

```diff
@@ -719,6 +727,176 @@ public Limits setUserLimits(@ApiParam(hidden = true) @Auth User authUser,
         return getStrippedServices(userDAO.findById(user.getId()));
     }
 
+    @POST
+    @Timed
+    @UnitOfWork
+    @Path("/workflows")
+    @Operation(operationId = "addWorkflow")
```

Comment:
What distinguishes this from `manualRegister`?

Reply:
Hm that is a good point, could probably just call manual register and pass it some default values.

Reply:
Would it makes sense to just use the manualRegister and pass default values in the UI, or should I just have the addWorkflow endpoint that calls manualRegister using the defaults?

Reply:
Either is reasonable, I would probably prefer the former and have good tests for the two cases (i.e. defaults set or not).